website/static/css/style.css

```diff
@@ -186,6 +186,16 @@ a {
         width: 100%;
         margin-bottom: 20px;
     }
+    /* Allow project nav to move the content on small screen */
```

Comment:
Am I correct in assuming that there is no shared project-specific CSS file to put these styles in?